### PR TITLE
Create Add_UserAgent_in_comment use case

### PR DESCRIPTION
**Pull request by Men-hau**

### CODE REVIEW

File: docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Enrich Alert with User agent in comment
+
+This use case describes how to enrich an alert with the User agent in comment
```
Comment (qbourgue) [inline docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md:3]:
```suggestion
This use case describes how to enrich the comments of an alert with the User agent extracted from the associated events.
```

Comment (qbourgue) [reply docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md:3]: A suggestion, it's up to you. However, if you keep your sentence, you should add the final dot.

File: docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	* `SIC_WRITE_ALERTS_COMMENT`
+
+!!! note
+    To create your API Key, follow this [documentation](../../../getting_started/generate_api_keys.md).
+
+## Playbook configuration
+
+Find the playbook configuration below: 
+
+![Playbook Add_UserAgent_in_comment](docs/assets/playbooks/library/UseCases/Add_UserAgent_in_comment.md.png)
```
Comment (qbourgue) [inline docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md:21]: The image is not displayed in the preview, is it normal?

File: docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+| Module | Configuration |
+| --- | --- |
+| Alert Created | configure module & trigger configuration |
+| Get_an_alert | uuid = `alert_uuid` of **Created alert** |
```
Comment (qbourgue) [inline docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md:26]:
```suggestion
| Alert Created | configure module & trigger configuration |
| Get Alert | uuid = `alert_uuid` of **Created alert** |
```

File: docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+`Get Event Field Common Values`
+  - **earliest_time**   `{{ ((node.2.first_seen_at | iso8601_to_timestamp) - 3600000) | timestamp_to_iso8601 }}`
+  - **fields**  `user_agent.original`
+  - **latest_time**    `now`    
+  - **query**   `source.ip:"{{ node.2['source'] }}" AND destination.ip:"{{ node.2['target'] }}" AND [entity.name](http://entity.name/):"{{ node.2['entity']['name'] }}"`
+    
+`post_a_comment_on_an_alert`
```
Comment (qbourgue) [inline docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md:38]: The format is not valid in the preview, you should apply changes

File: docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+Find the playbook configuration below: 
+
+![Playbook Add_UserAgent_in_comment](docs/assets/playbooks/library/UseCases/Add_UserAgent_in_comment.md.png)
+
+| Module | Configuration |
+| --- | --- |
+| Alert Created | configure module & trigger configuration |
+| Get_an_alert | uuid = `alert_uuid` of **Created alert** |
+| Get Event Field Common Values | See fields below |
+| post_a_comment_on_an_alert | See fields below |
```
Comment (qbourgue) [inline docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md:28]:
```suggestion
| Comment Alert | See fields below |
```

File: docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+`Get Event Field Common Values`
+  - **earliest_time**   `{{ ((node.2.first_seen_at | iso8601_to_timestamp) - 3600000) | timestamp_to_iso8601 }}`
+  - **fields**  `user_agent.original`
+  - **latest_time**    `now`    
+  - **query**   `source.ip:"{{ node.2['source'] }}" AND destination.ip:"{{ node.2['target'] }}" AND [entity.name](http://entity.name/):"{{ node.2['entity']['name'] }}"`
```
Comment (qbourgue) [inline docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md:36]:
```suggestion
  - **query**   `source.ip:"{{ node.2['source'] }}" AND destination.ip:"{{ node.2['target'] }}" AND entity.name:"{{ node.2['entity']['name'] }}"`
```

Comment (qbourgue) [reply docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md:36]: I think there is a wrong copy/paste in the query

Comment (qbourgue) [inline docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md:36]: Are you sure that this query allows in all case to retrieve events that contain a User agent? If not, you should specify that the query could vary according to the event types / intakes

**Review by qbourgue — CHANGES_REQUESTED** [review]

File: docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md (hunk 4 above)
Comment (Men-hau) [reply docs/xdr/usecases/playbook/Add_UserAgent_in_comment.md]: correction done on entity.name

**Review by qbourgue — APPROVED** [review]

**Pull request merged**